v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -69,6 +69,11 @@ export const ERROR_CATEGORIES: {
     max: 1199,
     websiteTitle: "Contract name errors",
   },
+  PLUGINS: {
```

Comment:
I added separate plugins section. I initially put these in general. Let me know if I should move it back.

Reply:
looks good